Modules/+Drivers/+Kinesis/KinesisBSC203.m

```diff
@@ -0,0 +1,347 @@
+classdef KinesisBSC203 < Drivers.Kinesis.Kinesis_invisible & Modules.Driver
+    % Driver for kinesis stages. Currently only works with motor_channels = [1 2 3] (i.e. x y z motors in ascending order). Situations other than this will result in wrong motors moving, to be fixed.
+    properties
+        name
+        motor_channels
+    end
+    properties(Constant, Hidden)
+        GENERICMOTORDLL='Thorlabs.MotionControl.GenericMotorCLI.dll';
+        GENERICMOTORCLASSNAME='Thorlabs.MotionControl.GenericMotorCLI.GenericMotorCLI';
+        STEPPERMOTORDLL='Thorlabs.MotionControl.Benchtop.StepperMotorCLI.dll';
+        STEPPERMOTORCLASSNAME='Thorlabs.MotionControl.Benchtop.StepperMotorCLI.BenchtopStepperMotor';
+    end
+
+    properties(SetAccess = private, SetObservable, AbortSet)
+        isconnected = false;         % Flag set if device connected
+        serialnumbers;               % Device serial numbers
+        controllername;              % Controller Name
+        controllerdescription        % Controller Description
+        stagename;                   % Stage Name
+        acceleration;                % Acceleration
+        maxvelocity;                 % Maximum velocity limit
+        minvelocity;                 % Minimum velocity limit
+        positions = [NaN, NaN, NaN];                   % Motor position (1 * 3 array)
+
+        Homed;
+        isMoving = false;
+
+        Travel = [-2 2] * 1000;
```

Comment:
Is this hardcoded for all kinesis stages?

Reply:
We didn't fully understand what was going on here, but there seemed to be a mysterious factor of 2 between the commands that are sent in and the actual motion of the stage. As far as we know it's the same from device to device. Might be good to test on another setup, but in principle calibration could be handled by the stage calibration if it is different.

Reply:
I've now taken out the default value for calibration and travel. The travel is a required input for this BSC203 stage during initialization. I've changed the 'calibration' property name to 'factor' and made it a pref that the user can set manually depending on how the stage is responding